lib/language.js

```diff
@@ -2016,6 +2038,7 @@ function init() {
       ,ru: 'Стереть запись'
       ,sk: 'Zmazať záznam'
       ,nl: 'Verwijder invoer'
+      ,ko: '삭제'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2737942153/issues/source?bid=2452379&fileBranchId=3352843#l2041)